Assets/Scripts/Notebook/Notebook Interactions/ColorController.cs

```diff
@@ -7,6 +7,24 @@
 /// </summary>
 public class ColorController : MonoBehaviour
 {
+    private static ColorController _colorController;
+    //singleton instance
+    public static ColorController Instance
+    {
+        get
+        {
+            if (!_colorController)
+            {
+                _colorController = FindObjectOfType(typeof(ColorController)) as ColorController;
+                if (!_colorController)
+                {
+                    UnityEngine.Debug.LogError("There needs to be one active ColorController script on a GameObject in your scene (usually on right hand anchor).");
+                }
+            }
+            return _colorController;
+        }
+    }
```

Comment:
I don't think this should be a singleton. What if you wanted to add a color controller to the left hand for some reason? Then you wouldn't be able to.

Reply:
yee